pcsx2-qt/AutoUpdaterDialog.cpp

```diff
@@ -393,7 +392,7 @@ void AutoUpdaterDialog::getChangesComplete(QNetworkReply* reply)
 					   "that you will have to reconfigure your settings after this update.</p>"));
 			}
 			changes_html += tr("<h4>Installing this update will download %1 MB through your internet connection.</h4>")
```

Comment:
If we want to be pedantic and change this to 1024 instead of 1000, should make this `MiB`, not MB.

Reply:
I changed it to 1024 to match GitHubs UI, which also reports it (incorrectly) as MB.

Reply:
Alright, fair enough.